src/typings.d.ts

```diff
@@ -25,6 +25,7 @@ declare enum StepStates {
 
 interface ProgressStep {
   label: string;
+  subtitle: string;
```

Comment:
I think the subtitle should be optional.

Reply:
Agreed